### ISSUE TITLE
webpack-dev-server error

### ISSUE THREAD

**Issue by bmbrina** [shackijj/widget-typescript#1]
There's an error when creating a new template from scratch:

```
Build failed!
✖ ERROR ../node_modules/webpack-dev-server/client/utils/createSocketUrl.js
Module not found: Error: Can't resolve 'url' in './node_modules/webpack-dev-server/client/utils'
 @ ../node_modules/webpack-dev-server/client/utils/createSocketUrl.js 4:10-24
 @ ../node_modules/webpack-dev-server/client?http://0.0.0.0:8080
 @ multi ../node_modules/webpack-dev-server/client?http://0.0.0.0:8080 ../node_modules/webpack/hot/dev-server.js ../node_modules/preact-cli/lib/lib/entry webpack-dev-server/client webpack/hot/dev-server
```

**Steps to reproduce**
1. Run `preact create widget-typescript my-project`
2. Run `npm install`
3. Run `npm run dev`

**Additional Information**
It only impacts development and not build

**Comment by reznord** [comment]
Hi @bmbrina, this is an issue with native-url being in devDeps instead of deps of preact-cli package.

I will be publishing an RC shortly to fix this.

**Comment by nikkieverett** [comment]
I found that deleting the alias for `url: 'native-url'` within the preact.config.js file resolved the error.

**Comment by reznord** [comment]
This should be fixed in [3.0.0-rc.14](https://github.com/preactjs/preact-cli/releases/tag/3.0.0-rc.14).

Feel free to re-open if the issue still persists 😄
